store/tikv/coprocessor.go

```diff
@@ -70,7 +70,7 @@ func supportExpr(exprType tipb.ExprType) bool {
 	case tipb.ExprType_JsonType, tipb.ExprType_JsonExtract, tipb.ExprType_JsonUnquote, tipb.ExprType_JsonValid,
 		tipb.ExprType_JsonObject, tipb.ExprType_JsonArray, tipb.ExprType_JsonMerge, tipb.ExprType_JsonSet,
 		tipb.ExprType_JsonInsert, tipb.ExprType_JsonReplace, tipb.ExprType_JsonRemove, tipb.ExprType_JsonContains:
-		return false
+		return true
```

Comment:
json_valid and json_contains are not implemented in tidb either. So you only need to implement json_remove now.